cli_tools/common/utils/storage/object_handle.go

```diff
@@ -0,0 +1,81 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package storage
+
+import (
+	"context"
+	"io"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/domain"
+)
+
+// ObjectHandleCreator is responsible for creating GCS Object Handle.
+type ObjectHandleCreator struct {
+	ctx context.Context
+	sc  *storage.Client
+}
+
+// GetObject gets GCS object interface.
+func (ohc *ObjectHandleCreator) GetObject(
+	bucket string, object string) domain.StorageObjectInterface {
+	return &ObjectHandle{
+		ob: ohc.sc.Bucket(bucket).Object(object), ctx: ohc.ctx}
+}
+
+// ObjectHandle is a wrapper around storage.ObjectHandle. Implements StorageObjectInterface.
+type ObjectHandle struct {
```

Comment:
Why not name this struct `StorageObject`? 

Several issues having it named `ObjectHandle` 
- it overlaps with storage.ObjectHandle so it's not clear on first look which one is being referred to.
- ObjectHandle sounds like it's a simple reference without much logic, while StorageObjectInterface provides a lot of functionality on top of the handle.
- having confusing calls such as `objectHandle.GetObjectHandle()`

Reply:
That makes sense, I will change the name to StorageObject.